ICSharpCode.Decompiler/IL/ControlFlow/SymbolicExecution.cs

```diff
@@ -118,7 +118,7 @@ public void AddStateVariable(ILVariable v)
 
 		public SymbolicValue Eval(ILInstruction inst)
 		{
-			if (inst is BinaryNumericInstruction bni && bni.Operator == BinaryNumericOperator.Sub && !bni.CheckForOverflow)
+			if (inst is BinaryNumericInstruction bni && bni.Operator == BinaryNumericOperator.Sub)
```

Comment:
I don't think this change is safe. Maybe we need an extra flag switching symbolic evaluation to a "VB mode"?